### DISCUSSION TITLE
Modulo 1 , TypeError :Traceback (most recent call last) (RESUELTO)

### DISCUSSION THREAD

**Discussion by AlexAntonioG** [LaunchX-InnovaccionVirtual/CursoIntroPython#146] [Q&A] [answered]
Que tal!, estaba checando la documentación del modulo uno y pues en los correspondientes bloques de código iba realizando los ejercicios como se va indicando, hasta que llegue a la parte de las _Entradas(Input)_ y me marca el error **_TypeError :Traceback (most recent call last)_** , adjunto una imagen, no me deja pasar esa parte del código. ¿A que creen que se deba?

![imagen](https://user-images.githubusercontent.com/68760251/153661789-9f9c492e-cca9-43cc-a603-0019bbcb735f.png)

**Comment by miguelapol** [comment]
Hola se debe cuando imprimes el nombre en este caso usa el "," en ves del +
print('Saludo',nombre)

**Comment by AlexAntonioG** [comment, marked as answer]
**EDIT: Ya he resuelto el problema, si a alguien más le sirve, solo deben reiniciar el Kernel y listo**
![imagen](https://user-images.githubusercontent.com/68760251/153663274-2d99db2e-49cd-491c-8a83-071d3e270064.png)

**Comment by AlexString** [comment]
Hola! Que bueno que pudiste terminar tu Kata 1 Ahora solo quiero realizar una pequeña aclaración.
Respecto a lo que dice @miguelapol.

Esto no es del todo cierto, de hecho, es indiferente, python puede concatenar cadenas de caracteres con +
o también la función print() puede recibir argumentos:

![image](https://user-images.githubusercontent.com/78276022/153663926-dffc163c-d322-4b61-937b-83b77432f977.png)
